app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/VatRegisteredUkController.scala

```diff
@@ -102,22 +102,24 @@ class VatRegisteredUkController @Inject() (
                 if (isInReviewMode)
                   if (yesNoAnswer.isYes)
                     Future.successful(Redirect(VatDetailsController.reviewForm(service).url))
-                  else {
-                    subscriptionDetailsService.clearCachedUkVatDetails
-                    Future.successful(Redirect(DetermineReviewPageController.determineRoute(service).url))
-                  }
+                  else
+                    subscriptionDetailsService.clearCachedUkVatDetails flatMap { _ =>
```

Comment:
For this, do we want to store nothing in the cache or should we store and display the fact that the user has answered no? My assumption is its probably the latter. Lets jump on a call to discuss as I might have misinterpreted.

Reply:
Hi Rob, the changes here affect these values in the CYA page - this is an example of what shows in main branch currently if the user selects 'No' - this should look the same in this branch if the user selects Yes then goes back and selects No - Happy to have a call if I'm missing something though :)

<img width="685" alt="Screenshot 2022-11-02 at 08 47 33" src="https://user-images.githubusercontent.com/12535866/199442194-dd3f32fc-f152-4471-b6e8-ff01aeee52c1.png">